public/i18n/qqq.json

```diff
@@ -19,5 +19,8 @@
 	"query-builder-result-error-incomplete-form": "Error message when the form is incomplete",
 	"query-builder-result-error-missing-property": "Error when property is missing",
 	"query-builder-result-error-missing-value": "Error when value is missing",
-	"query-builder-add-condition": "Text on the button that adds a condition"
+	"query-builder-add-condition": "Text on the button that adds a condition",
+	"query-builder-valuetyperelation-dropdown-matching": "Text on dropdown option: matching",
+	"query-builder-valuetyperelation-dropdown-without": "Text on dropdown option: without",
```

Comment:
These could maybe be made more descriptive from the translator's point of view. We could extend them to say something like `Text on dropdown option to have the Query Builder select items that have the given property, but not with the given value. See also {{msg-mw|query-builder-valuetyperelation-dropdown-matching}} and {{msg-mw|query-builder-valuetyperelation-dropdown-regardless-of-value}}` and similarly for the other two

Reply:
Done!